config/features/flags.go

```diff
@@ -46,6 +46,10 @@ var (
 		Name:  "save-invalid-block-temp",
 		Usage: "Writes invalid blocks to temp directory.",
 	}
+	saveInvalidBlobTempFlag = &cli.BoolFlag{
+		Name:  "save-invalid-blob-temp",
+		Usage: "Writes invalid blobs to temp directory.",
```

Comment:
Should this mention "gossip" since this doesn't apply to init-sync or backfill? (Same q for `--save-invalid-block-temp`)